src/statsig/constants.ts

```diff
@@ -25,6 +25,7 @@ export const LayerParams = {
 export const FeatureGates = {
   [StatsigFeatureGates.USE_ZENDESK_API_FOR_SUPPORT]: false,
   [StatsigFeatureGates.SHOW_POSITIONS]: false,
+  [StatsigFeatureGates.SHOW_DAPP_SHORTCUTS]: false,
```

Comment:
nit: more specific so we can add new gates for more advances shortcuts in the future
```suggestion
  [StatsigFeatureGates.SHOW_CLAIM_SHORTCUTS]: false,
```

Reply:
updated